notes/returned-mail-received.vbs

```diff
@@ -166,7 +202,7 @@ IF mailing_address_confirmed = "YES" or residential_address_confirmed = "YES" TH
     '-------------------------------------------------------------------------------------------------DIALOG
     Dialog1 = "" 'Blanking out previous dialog detail
     IF ADDR_actions = "no forwarding address provided" THEN
-        BeginDialog Dialog1, 0, 0, 186, 215, "no forwarding Address Provided"
+        BeginDialog Dialog1, 0, 0, 186, 215, "RETURNED MAIL PROCESSING NO FORWARDING ADDRESS PROVIDED"
           CheckBox 10, 95, 165, 10, "Verif Request (DHS-2919A) Request for Contact", verif_request_checkbox
```

Comment:
I am unclear on these verification checkboxes. I don't know the intention here. If we are documenting a verification request, we should be really clear that the worker has requested verification and what verification is being requested. Additionally, we would never request a SVF or CRF, we might request address information or shelter expense verification. I am not sure what 'Returned Mail/Other' is referring to.